cool-core/src/test/java/com/nus/cool/core/model/CoolModelTest.java

```diff
@@ -62,6 +79,29 @@ public void CubeListTest() throws IOException {
         System.out.println("Applications: " + Arrays.toString(cubes2));
     }
 
+    @Test (priority = 10)
+//show the CUBE list is empty
+    public void CubeEmptyListTest() throws IOException {
+        System.out.println("======================== Cube List Test ========================");

```

Comment:
Please update this log.

Reply:
but I do not think that we should add this test function. It only shows a minor difference from the previous one.

Reply:
agree,thanks